controllers/namespacelabel/namespacelabel_controller.go

```diff
@@ -278,7 +283,7 @@ func Uninstall(v string, request reconcile.Request, r *ReconcileNamespaceLabel)
 
 	if rhmiCr.DeletionTimestamp == nil {
 		r.log.Info("Deleting RHMI CR")
-		err := r.client.Delete(r.context, rhmiCr)
+		err := r.Delete(context.TODO(), rhmiCr)
```

Comment:
Why have we converted back to using context.TODO instead of the passed around context?

Reply:
If I remember correctly, we stopped storing the context in structs as part of the work to fix the memory leak a while ago. I think the change was originally missed here

Reply:
Yeah, we shouldn't be storing the context, but it should still be passed around from the rhmi controller, just not stored anywhere. Creating new ones everywhere means that we lose the ability to abort a reconcile if required.

Reply:
e.g. at the beginning of the uninstall reconcile, we abort any - potential - running install reconcilers. This was to address a bug where the uninstall would go faster than a running install reconcile, and the still running install reconcile would put everything back after it got deleted.

Reply:
I've just noticed that this is in the namespace label controller. So this can be ignored. So long as all the product reconcilers are still being passed a context from the installation controller.

Reply:
actually, this information appears to be an out of date, so disregard this thread.